jerry-core/debugger/jerry-debugger-ws.c

```diff
@@ -13,8 +13,6 @@
  * limitations under the License.
  */
 
-#ifdef JERRY_DEBUGGER
```

Comment:
What is the reason for this change? #include <arpa/inet.h> will surely fail with jerry_libc, and we need to think about all-in-one builds.